src/Products/SimpleProductEntity.php

```diff
@@ -1,13 +1,14 @@
 <?php
 namespace FurnitureStoreApi\Products;
+use FurnitureStoreApi\Services\CurrencyConversionClass;
 use FurnitureStoreApi\Services\HeaderService;
 HeaderService::setHeaders();
 
 class SimpleProductEntity implements \JsonSerializable
 {
     protected int $id;
     protected string $name;
-    protected float $price;
+    protected string $price;
```

Comment:
Why did you change this to a string?

Reply:
That was feedback from a different Story, but happy to change it

Reply:
I think I get it.